### PR TITLE
Aniket1743

### PR DESCRIPTION
**Pull request by aniket1743**
Added functions to fetch data from db.

### CODE REVIEW

File: client/src/components/GetCaption.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
+import React, { Component } from 'react'
+import axios from 'axios';
+class GetCaptionComponent extends Component {
```
Comment (sahil505) [inline client/src/components/GetCaption.js]: we are using react-hooks and not classes. Can you please make changes to this to implement react-hooks?

File: client/src/App.js [hunk 2]
```diff
@@ -8,6 +9,7 @@ const App = () => (
       <i className="fas fa-music"></i> Cap That Pic
     </h4>
 
+    <GetCaptionComponent />
```
Comment (sahil505) [inline client/src/App.js]: Needs to be after the FileUpload component.

File: client/src/App.js [hunk 3]
```diff
@@ -1,4 +1,5 @@
 import React from "react";
+import GetCaptionComponent from './components/GetCaption';
```
Comment (sahil505) [inline client/src/App.js]: better naming `GetCaption` rather than `GetCaptionComponent`?

File: client/src/App.js (hunk 2 above)
Comment (aniket1743) [reply client/src/App.js]: It is just temporary thing to check the connection between Frontend and Backend. I will remove the whole file after checking.

File: client/src/components/GetCaption.js (hunk 1 above)
Comment (aniket1743) [reply client/src/components/GetCaption.js]: Tenporaray thing.

Comment (sahil505) [reply client/src/components/GetCaption.js]: Okay

**Pull request merged**